_components/dropdown/guidance/when-to-consider-something-else.md

```diff
@@ -1,4 +1,4 @@
 - **Fewer than seven options.** Use radio buttons instead.
-- **More than 15 options.** If the list of options is very long. Let users type the same information into a text input that suggests possible options instead.
+- **More than 15 options.** If the list of options is very long, consider using a [combo box]({{ site.baseurl }}/components/combo-box), or let users type the same information into a text input.
```

Comment:
Maybe this should strike the "text input" recommendation altogether, if the idea is there'd be a predefined set of options?

```suggestion
- **More than 15 options.** If the list of options is very long, consider using a [combo box]({{ site.baseurl }}/components/combo-box).
```